packages/coinstac-ui/app/render/state/graphql/props.js

```diff
@@ -27,8 +27,9 @@ export const consortiaMembershipProp = (name) => {
             const consQuery = store.readQuery({ query: FETCH_ALL_CONSORTIA_QUERY });
             const index = consQuery.fetchAllConsortia.findIndex((cons) => {
               if (cons && data[name]) {
-                cons.id === data[name].id;
+                return cons.id === data[name].id;
```

Comment:
This changes program behavior! Could this be a bug where `index` is always `-1`?

Reply:
Great catch! I wonder... there are quite a few odd UI bugs

Reply:
Thank the magic of ESLint 🔮